cfe_internal/enterprise/federation/federation.cf

```diff
@@ -410,6 +410,27 @@ body contain contain_transport_user
     useshell => "true";
 }
 
+bundle monitor measure_psql
+{
+  measurements:
```

Comment:
Oh, i nearly forgot. This hsould be guarded so that it does not try to activate on any other host. Monitor bundles like server bundles don't need to be in the bundlesequence, so they need to be carefully guarded so as not to apply on all hosts.

Reply:
So id do am_policy_hub| policy_server and maybe if => fileexists( "/var/log/postgresql.log" )

Reply:
I tried this with `am_superhub::` but that didn't seem to work, maybe due to the policy being evaluated differently in cf-monitord?

Reply:
Yeah, as we discussed in chat, I think because am_superhub is defined from an agent bundle AND it's not a persistent class so monitor bundles won't know about it.